silhouette/test/com/mohiva/play/silhouette/impl/providers/CredentialsProviderSpec.scala

```diff
@@ -107,15 +121,29 @@ class CredentialsProviderSpec extends PlaySpecification with Mockito {
     lazy val fooHasher = {
       val h = mock[PasswordHasher]
       h.id returns "foo"
+      h.isSuitable(any) answers { p =>
```

Comment:
These two hashers and the registry are exactly the same as in the `BasicAuthProviderSpec`, can't they be reused from a common location?